src/ext/java/org/opentripplanner/ext/vectortiles/layers/vehiclerental/mapper/DigitransitRealtimeVehicleRentalStationPropertyMapper.java

```diff
@@ -0,0 +1,37 @@
+package org.opentripplanner.ext.vectortiles.layers.vehiclerental.mapper;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Locale;
+import java.util.stream.Collectors;
+import org.opentripplanner.common.model.T2;
+import org.opentripplanner.ext.vectortiles.I18NStringMapper;
+import org.opentripplanner.ext.vectortiles.PropertyMapper;
+import org.opentripplanner.routing.vehicle_rental.VehicleRentalStation;
+
+public class DigitransitRealtimeVehicleRentalStationPropertyMapper
+  extends PropertyMapper<VehicleRentalStation> {
+
+  private final I18NStringMapper i18NStringMapper;
+
+  public DigitransitRealtimeVehicleRentalStationPropertyMapper(Locale locale) {
+    this.i18NStringMapper = new I18NStringMapper(locale);
+  }
+
+  @Override
+  protected Collection<T2<String, Object>> map(VehicleRentalStation station) {
+    return List.of(
+      new T2<>("id", station.getId().toString()),
```

Comment:
It might only be a 3 lines of code but could you please extract a reusable method for the realtime and non-realtime mappers?

Reply:
Form factors should also be in there.

Reply:
Perhaps you want to use `ListUtils.combine` for this.

Reply:
Hmm, perhaps the immutable lists are slower, not sure. If so, then use a mutable one.

Reply:
I did something in https://github.com/opentripplanner/OpenTripPlanner/pull/4529/commits/fa058cbc2e4323281393c0dce49cf9cc2b5bdd27 . Didn't share any code with the legacy rental mapper because it doesn't use the feedScopedId, I think we can hopefully get rid of that mapper at some point anyway.

Reply:
There were some performance issues, which resulted in the current structure, but I don't know if this is an issue for the vehicle rentals